.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -65,73 +53,66 @@ assignees: ''
       * peer-pods: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/peer-pods/kustomization.yaml
           Note that we need the quay.io/confidential-containers/runtime-payload-ci registry and kata-containers-latest tag
 
-8. - [ ] Update peer-pods with latest commits of kata-containers and attestation-agent and test it, following the [release candidate testing process](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#release-candidate-testing)
+6. - [ ] Update peer-pods with latest commits of kata-containers and attestation-agent and test it, following the [release candidate testing process](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#release-candidate-testing)
     
-9. - [ ] Cut an ocicrypt-rs v<TARGET_RELEASE> release, if changes happened in the project
-
-10. - [ ] Cut an attestation-agent v<TARGET_RELEASE>, if changes happened in the project
-
-11. - [ ] Cut an attestation-service v<TARGET_RELEASE> and make images for AS and RVPS, if changes happened in the project.
+7. - [ ] Cut an attestation-service v<TARGET_RELEASE> and make images for AS and RVPS, if changes happened in the project.
 
        * https://github.com/confidential-containers/attestation-service
        * Cut a release (AS/RVPS images will be automatically built triggered by release)
 
-12. - [ ] Update kbs to use the latest commit from attestation-service, cut a release and make image
+8. - [ ] Update kbs to use the latest commit from attestation-service, cut a release and make image
```

Comment:
This actually has to happen after step 9, because we also need to update [this](https://github.com/confidential-containers/kbs/blob/main/tools/client/Cargo.toml#L16).

Reply:
@fitzthum 

I was going to make change but I realized one thing... don't we need to update the kbs twice as the other components?

I mean, on first pass we update the kbs to the latest commit of attestation-service and guest-components  *before* cutting a release of those components, just like we do on step 2. Then we test kbs. The 2nd pass should happen after the attestation-service and guest-components releases, this time we use the tags.

Reply:
Yes, in theory we should use the rc version of the guest components and then update it again with the final version, but note that this dependency is just for the client tool and is only pulling in the KbsProtocol. I'm not sure if the KBS CI even covers the client tool. So for now I think it might be overkill to do two versions.

Reply:
@fitzthum updated. See if it is better now and I captured the step correctly, please.